inductiva/tasks/methods.py

```diff
@@ -51,51 +48,6 @@ def _fetch_tasks_from_api(
             raise e
 
 
-def _dataframe_of_tasks(tasks: Sequence["inductiva.tasks.Task"]) -> str:
-    columns = [
-        "ID", "Simulator", "Status", "Submitted", "Started", "Computation Time",
-        "Resource Type"
-    ]
-    rows = []
-
-    for task in tasks:
-        info = task.get_info()
-        simulator = task.get_simulator_name()
-        status = task.get_status()
-
-        computation_end_time = info.get("computation_end_time", None)
-
-        execution_time = task.get_computation_time(fail_if_running=False)
-        if execution_time is not None:
-            if computation_end_time is None:
-                if status in ["started", "submitted"]:
-                    execution_time = f"*{execution_time}"
-                else:
-                    execution_time = "n/a"
-
-        executer = info["executer"]
-        if executer is None:
-            resource_type = None
-        else:
-            resource_type = executer["vm_type"]
-            if executer["n_mpi_hosts"] > 1:
-                resource_type += f" x{executer['n_mpi_hosts']}"
-
-        row = [
-            task.id,
-            simulator,
-            status,
-            format_utils.datetime_formatter(info.get("input_submit_time",
-                                                     None)),
-            format_utils.datetime_formatter(info.get("start_time", None)),
-            execution_time,
-            resource_type,
-        ]
-        rows.append(row)
-
-    return pd.DataFrame(rows, columns=columns)
-
-
 # pylint: disable=redefined-builtin
 def list(last_n: int = 5,
```

Comment:
@sergiomsantos can you comment on this? Should the list() return a list of objects (since that is what the get does) or  readable information about the tasks?

Reply:
It should only return a list of tasks because that is what the method suggests. If the user wants to have a pretty print of the list, he can build his own dataframe. I don't see why we should provide a customized `str` representation of a list of tasks at this point. What we should consider is how to provide a better  `str` representation of a task, but that falls outside the scope of this task.

Reply:
Indeed, but notice that this method isn't actually doing anything as is. What I was suggesting is to deliver a dictionary with all of this information, not the string of a list.